mappings/net/minecraft/client/input/Keyboard.mapping

```diff
@@ -0,0 +1,54 @@
+CLASS cvj net/minecraft/client/input/Keyboard
+	FIELD a client Lcvl;
+	FIELD b allowKeybinds Z
+	FIELD c clipboard Lctq;
+	FIELD d debugCrashStart J
+	FIELD e debugCrashElapsed J
+	FIELD f debugCrashStep J
+	FIELD g switchF3State Z
+	METHOD <init> (Lcvl;)V
+		ARG 1 client
+	METHOD a getClipboardContents ()Ljava/lang/String;
+	METHOD a (IJ)V
+		ARG 1 error
+		ARG 2 desc
+	METHOD a setCallbacks (J)V
+		ARG 1 window
+	METHOD a charTyped (JII)V
+		ARG 1 handle
+		ARG 3 codepoint
+		ARG 4 mods
+	METHOD a keyPressed (JIIII)V
+		ARG 1 window
+		ARG 3 key
+		ARG 4 scancode
+		ARG 5 action
+		ARG 6 mods
+	METHOD a putBlockOnClipboard (Lbvl;Lev;Lib;)V
```

Comment:
Would it make sense for these names to match with `setClipboardContents`?

Reply:
*`returnToClipboard` Whilst we're at it.

Reply:
Is it coming back from being on the clipboard already?

Reply:
Well the name is currently `putBlockOnClipboard` so I guess?

Reply:
It sets the clipboard contents

Reply:
Yeah, then it should be `setClipboardContents`. Maybe I misinterpreted gegy's comment, because it seemed like he was implying there was already another method with that name.